src/schemas/components/SchemaFormPropertyAnyOf.vue

```diff
@@ -73,10 +72,10 @@
     const selectedProperty = props.property.anyOf[selectedPropertyIndex.value]
 
     if (isPropertyWith(selectedProperty, '$ref')) {
-      return getSchemaDefinition(schema, selectedProperty.$ref)
+      return merge({}, getSchemaDefinition(schema, selectedProperty.$ref), props.property)
     }
 
-    return selectedProperty
+    return merge({}, selectedProperty, props.property)
```

Comment:
These merges might be backwards - I think property definitions from a selected property should overwrite those of the base property, so props like title and description don't get displayed incorrectly

Reply:
So what's happening here is for the property label and description we want to always show the value from the property itself. Otherwise in a union of classes you'll never see the property's name. You'll just see "User", when the property itself might be called "Owner". 

Then you see the different available types in the button group. So you can click "User" or possibly something like "None". 

![image](https://github.com/PrefectHQ/prefect-ui-library/assets/6200442/b48de265-158b-4685-b146-6a21552f5e0b)